shell/platform/windows/flutter_window.cc

```diff
@@ -66,11 +68,62 @@ static HCURSOR GetCursorByName(const std::string& cursor_name) {
   return ::LoadCursor(nullptr, idc_name);
 }
 
+static constexpr int32_t kDefaultPointerDeviceId = 0;
+
+// This method is only valid during a window message related to mouse/touch
+// input.
+// See
+// https://docs.microsoft.com/en-us/windows/win32/tablet/system-events-and-mouse-messages?redirectedfrom=MSDN#distinguishing-pen-input-from-mouse-and-touch.
+static FlutterPointerDeviceKind GetFlutterPointerDeviceKind() {
```

Comment:
nit: For static functions (as opposed to class methods), for conistency, we should probably move these to the anonymous namespace instead of old-school static.